lib/vsc/filesystem/lustre.py

```diff
@@ -625,3 +638,32 @@ def _set_quota(self, who, obj, typ=Typ2Opt.user, soft=None, hard=None, inode_sof
         opts.append(obj)
 
         self._execute_lfs('setquota', opts, True)
+
+
+    def xdmod_list_quota(self, devices=None):
```

Comment:
looks like a lot of copy paste compared to gpfs one. can you factor out the common bits?

Reply:
yes, i definitely should move this, this is even exactly the same for now. But I'm not sure it should be part of the posix class, since it won't work for all posix of course. But stuff is missing that is not really part of the vsc-filesystem scope, so maybe this should even be moved to another module.

Reply:
@itkovian any objections to move this in vsc-reporting? it will add vsc-filesystems as a dependency though

Reply:
@kwaegema you need to make producer simliar to https://github.ugent.be/hpcugent/vsc-cloud/blob/master/lib/vsc/cloud/xdmod/producer.py or https://github.ugent.be/hpcugent/vsc-reporting/blob/master/lib/vsc/reporting/xdmod/slurm.py